frontend/src/components/Image/styles.js

```diff
@@ -0,0 +1,25 @@
+import {StyleSheet} from 'aphrodite';
+
+export default StyleSheet.create({
+  cross_box_colour: {
+    background: "tomato" // "linear-gradient( to top, #49CF87, #40C080)"
+  },
+  image_wrapper: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    flexWrap: "wrap"
+  },
+  image_box: {
+    maxWidth: "200px",
+    maxHeight: "200px"
+  },
+  close_button: {
+    position: "absolute",
+    top: "0px",
```

Comment:
`top: 0,` also works

Reply:
yes but trying to keep things consistent. I understand above you say special case in aphrodite, but I think it's just easier to use the same everywhere, it's only a coupla chars.